probe/endpoint/ebpf/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/weaveworks/scope/probe/endpoint"
+)
+
+func main() {
+	tr := endpoint.NewEbpfTracker("/home/asymmetric/code/kinvolk/bcc/examples/tracing/tcpv4tracer.py")
```

Comment:
hardcoded path.

This file should be converted to Go tests and add some real tests.